drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -270,7 +270,10 @@ Error AudioDriverWASAPI::audio_device_init(AudioDeviceWASAPI *p_device, bool p_c
 	}
 
 	hr = p_device->audio_client->Initialize(AUDCLNT_SHAREMODE_SHARED, streamflags, p_capture ? REFTIMES_PER_SEC : 0, 0, pwfex, NULL);
-	ERR_FAIL_COND_V(hr != S_OK, ERR_CANT_OPEN);
+	if (hr != S_OK) {
+		ERR_PRINTS("WASAPI: Initialize failed with error 0x" + String::num_int64(hr, 16));
```

Comment:
You can use `ERR_EXPLAIN`, so that it shows only this message and not the default one that `ERR_FAIL_V` would raise.

Reply:
(i.e. `ERR_EXPLAIN` gives the error message to show in the following `ERR_FAIL`.)

Reply:
Just changed it